pinc/forum_interface.inc

```diff
@@ -636,7 +652,7 @@ function get_topic_details($topic_id)
         INNER JOIN $phpbb_users ON $phpbb_topics.topic_poster = $phpbb_users.user_id
         WHERE $phpbb_topics.topic_id = $topic_id
```

Comment:
sprintf?

Reply:
Sure, I can do this, but it makes for a very difficult to read
format string. Given `phpbb_topics`, `phpbb_forums`, `php_users` are local variables, I'm not sure if it's worth it. What do you think?